### ISSUE TITLE
ValueError: Unicode strings

### ISSUE THREAD

**Issue by tch1001** [xyz2tex/svg2tikz#79]
When running
```python
import svg2tikz
svg2tikz.convert_svg("sample.svg", verbose=True)
```
I get this error
```
ValueError: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
```
Any help would be appreciated!

**Comment by ldevillez** [comment]
Hello @tch1001 :),
Could you provide me a simple file leading to errors ?

**Comment by tch1001** [comment]
![sample](https://user-images.githubusercontent.com/17665556/216892703-3012e4f2-d63b-46fa-9517-6e3145d6c181.svg)
Thanks for your attention on this! This was the sample file that I had created using Inkscape!

**Comment by ldevillez** [comment]
Hello @tch1001,

I've checked your file and I see on the first line that the encoding is `"UTF-8"`. If you removed this part from the file (you can do it with a file editing software like notepad++ for example).

But I will check in the code if it is possible to allow `UTF-8`encoding :)

**Comment by ldevillez** [comment]
Fixed by #80 
(Pypi is not updated yet)
